tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -772,6 +772,44 @@ def forward_xp(self, inputs, xp):
         return tuple(b)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('shape,indices_or_sections', [
+    ((2, 4, 6), [1, 2, 4]),
+    ((2, 4, 6), [2, 4, 6]),
+    ((2, 4, 6), [2, -3, -5]),
+    ((2, 4, 6), [2, 8, 10]),
+    ((2, 4, 6), [4, 2, 6]),
+    ((2, 4, 6), [1, 3, 5]),
+    ((2, 4, 5), [1, -3, -5]),
+    ((2, 4, 8), [2, 5, 7, 6]),
+    ((2, 4, 10), [1, 4, 6, 8]),
+    ((2, 4, 6), numpy.array([1, 2, 3])),  # indices with 1-d numpy array
+    ((2, 4, 6, 8), numpy.array([1, 2, 3, 4])),  # indices with (4,)-shape numpy array
+    ((2, 4, 8, 10), numpy.array([1, 2, 3], numpy.int32)),
```

Comment:
Can you change the length of the first dimension (which vsplit splits over) to something larger, e.g. 6, 8 or 10 depending on the `indices_or_sections`?

Reply:
@hvy don't you think that will slow down the tests?

Reply:
How about reducing the lengths of the dimensions that are not of interest?